zsh-256color.plugin.zsh

```diff
@@ -19,7 +19,7 @@ _zsh_terminal_set_256color()
 
 	# Use (n-)curses binaries, if installed.
 	if [[ -x "$( which toe )" ]] ; then
-		if toe -a | egrep "^$TERM256" >/dev/null ; then
+		if toe -a | grep -e "^$TERM256" >/dev/null ; then
```

Comment:
The proper replacement for `egrep` is `grep -E`, I believe. At least the FreeBSD man page for grep(1) states:

```
     -E, --extended-regexp
             Interpret pattern as an extended regular expression (i.e., force
             grep to behave as egrep).
```

Reply:
Sounds good fixed in 2a3fabaaa0274d17f92f1aae3050f4898b7bca32